scio-core/src/main/scala/com/spotify/scio/values/SCollection.scala

```diff
@@ -1192,6 +1194,41 @@ sealed trait SCollection[T] extends PCollectionWrapper[T] {
   }
 
   /**
+    * Save this SCollection as a Cloud SQL database entry.
+    * @param writeOptions - option to create a Jdbc connection with database.
+    * @return Future tap with given type.
+    */
+  def saveAsCloudSql(writeOptions: JdbcWriteOptions[T]): Future[Tap[T]] = {
```

Comment:
Could you move it to the jdbc module? Take a look at how bigtable is handing this.